TUTORIAL.md

```diff
@@ -311,7 +315,7 @@ main = do
 
 ## Non-2XX responses
 
-By default, every request that generates a non-2XX response will generate a
+By default, every request that generates a non-2XX response will throw a
```

Comment:
Similarly here, let's just fix the offending content instead of correcting it later in the document.